From aac15d8d56857776bb0beb3d1e75b0f9b0bc8e4d Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:22:23 -0400
Subject: [redhat] kabi: Adding symbol add_wait_queue (kernel/wait.c)

Message-id: <6812b7517c9c96c733822d50548e1a51db554bb1.1410510773.git.poros@redhat.com>
Patchwork-id: 96016
O-Subject: [PATCH RHEL7.1 023/233] kabi: Adding symbol add_wait_queue (kernel/wait.c)
Bugzilla: 1071097
RH-Acked-by: Larry Woodman <lwoodman@redhat.com>

Bugzilla: 1071097
    https://bugzilla.redhat.com/show_bug.cgi?id=1071097

    Description:
    ============
    Kernel thread wait for event.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/add_wait_queue
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/add_wait_queue
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/add_wait_queue

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/add_wait_queue b/redhat/kabi/kabi-module/kabi_ppc64/add_wait_queue
new file mode 100644
index 0000000..2e5e521
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/add_wait_queue
@@ -0,0 +1,2 @@
+#1-
+0x735d8503 add_wait_queue vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/add_wait_queue b/redhat/kabi/kabi-module/kabi_s390x/add_wait_queue
new file mode 100644
index 0000000..7d7364d
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/add_wait_queue
@@ -0,0 +1,2 @@
+#1-
+0x1288f9c2 add_wait_queue vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/add_wait_queue b/redhat/kabi/kabi-module/kabi_x86_64/add_wait_queue
new file mode 100644
index 0000000..98b015a
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/add_wait_queue
@@ -0,0 +1,2 @@
+#1-
+0x5860aad4 add_wait_queue vmlinux EXPORT_SYMBOL
-- 
1.7.1